manage-apps/register.html

```diff
@@ -204,9 +206,15 @@
           $('#sandboxRegistration button[type="submit"]').attr('disabled', true);
         },
         success: function(data) {
-          console.log(data);
-          $("#sandboxRegistration").addClass('submited');
-          $('#form-success').removeClass('form-success-hidden');
+          if (data.sandboxErr !== 'No Error') {
```

Comment:
this part seems a little "magic stringy" to me - is there a more standard way to identify errors?